packages/components/addon/components/hds/pagination-bar/page-size.js

```diff
@@ -0,0 +1,21 @@
+import Component from '@glimmer/component';
+import { guidFor } from '@ember/object/internals';
+import { action } from '@ember/object';
+
+export default class HdsPaginationBarIndexComponent extends Component {
+  /**
+   * Generates a unique ID for the pageSize select
+   *
+   * @param pageSizeId
+   */
+  pageSizeId = 'pagination-page-size-' + guidFor(this);
```

Comment:
see? we were not "saving" that much time/code 🙂 (on the contrary we were introducing overrides/special cases in our design system)

Reply:
Sure I see.